### ISSUE TITLE
MacOS - 'The application "MyPaint" can't be opened'

### ISSUE THREAD

**Issue by kilometers** [mypaint/mypaint#894]
#### Behaviour

MyPaint won't start. Running from spotlight results in a window saying 'The application "MyPaint" can't be opened'. Running it from Terminal throws an error.

#### Basic system details

2017 Macbook Pro 13" 2.3 GHz 8GB RAM

MyPaint version: 1.2.1 and development 1.3.0 alpha (Both off of Macports)
Operating system: macOS 10.13.2
Desktop environment: n/a

#### Steps to reproduce

1. Install MyPaint (or MyPaint-devel) using Macports
2. Try to run MyPaint

I wish I had more info, but that's really all I've done. It's a new laptop, and I've not installed any other software packages aside from Node and React for javascript development.

#### Backtraces or error messages

```
INFO: mypaint: Installation layout: conventional POSIX-like structure with prefix u'/opt/local'
INFO: lib.i18n: OSX: imported AppKit.NSLocale OK. Will use for LANG, and for LANGUAGE order.
INFO: lib.i18n: OSX: LANG='en_US.UTF-8'
INFO: lib.i18n: OSX: LANGUAGE=u'en_US:zh_TW'
WARNING: mypaint: No bindtextdomain builtins found in module 'locale'.
INFO: mypaint: Trying platform-specific fallback hacks to find bindtextdomain funcs.
ERROR: mypaint: No platform-specific fallback for locating bindtextdomain is known for 'darwin'

(mypaint:1450): Gdk-CRITICAL **: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Traceback (most recent call last):
  File "/opt/local/bin/mypaint", line 464, in <module>
    main.main(datapath, iconspath, old_confpath, version=version)
  File "/opt/local/share/mypaint/gui/main.py", line 95, in main
    from gui import application
  File "/opt/local/share/mypaint/gui/application.py", line 53, in <module>
    import gui.device
  File "/opt/local/share/mypaint/gui/device.py", line 25, in <module>
    import gui.mode
  File "/opt/local/share/mypaint/gui/mode.py", line 817, in <module>
    class SingleClickMode (InteractionMode):
  File "/opt/local/share/mypaint/gui/mode.py", line 822, in SingleClickMode
    cursor = Gdk.Cursor.new(Gdk.CursorType.BOGOSITY)
TypeError: constructor returned NULL
```

**Comment by kilometers** [comment]
Installing MyPaint 2.0 resulted in the error from #634 

ERROR: mypaint: No platform-specific fallback for locating bindtextdomain is known for 'darwin'
Traceback (most recent call last):
  File "/opt/local/bin/mypaint", line 462, in <module>
    main.main(datapath, iconspath, old_confpath, version=version)
  File "/opt/local/share/mypaint/gui/main.py", line 92, in main
    lib.glib.init_user_dir_caches()
  File "/opt/local/share/mypaint/lib/glib.py", line 109, in init_user_dir_caches
    logger.debug("Init g_get_user_config_dir(): %r", get_user_config_dir())
  File "/opt/local/share/mypaint/lib/glib.py", line 71, in get_user_config_dir
    return filename_to_unicode(d_fs)
  File "/opt/local/share/mypaint/lib/glib.py", line 57, in filename_to_unicode
    ustring = GLib.filename_to_utf8(opsysstring, -1, 0, 0)
TypeError: GLib.filename_to_utf8() takes exactly 2 arguments (4 given)

**Comment by achadwick** [comment]
This was fixed in 2c07a27b00c421979cda17d7827fce85d276c28b. Please get your MacPorts build updated for 3.0.0-alpha.  This will not be fixed for the 1.2 series.

See also #750.
